doc/modules/linear_model.rst

```diff
@@ -814,7 +818,7 @@ each solver:
 +------------------------------+-----------------+-------------+-----------------+-----------+------------+
 
 The "saga" solver is often the best choice but requires scaling. The
-"liblinear" solver is used by default for historical reasons.
+"lbfgs" solver is used by default for historical reasons.
```

Comment:
Err... that's pretty wrong. We just moved to that solver. It's used because it's a great default!

Reply:
Ok i just changed the solver as the lbfgs explanation above plus the reference links say that it is good in some scenarios. So i need to update only that its being used as it is the best default.